.eslintrc.json

```diff
@@ -0,0 +1,40 @@
+{
+	"env": {
+		"browser": true,
+		"es6": true,
+		"webextensions": true
+	},
+	"extends": "eslint:recommended",
+	"globals": {
+		"chrome": true
+	},
+	"parserOptions": {
+		"ecmaVersion": 2018,
+		"sourceType": "module"
+	},
+	"rules": {
```

Comment:
Have you referred these rules from somewhere? Or wrote this by yourself?

Reply:
Obviously it's not gibberish, I used some of these in Smartprix assistant application, while for others I consulted ESLint docs

Reply:
If you want you can add your rules too @rajatgoyal715